wordpress-mobile/WordPress-iOS
# Make WebKitViewController handle links in new windows

First part of #8191, makes WebKitViewController handle links open in new windows. I'll do a second PR with some other improvements to pre-authenticate reader posts.

To test:

1. Go to the reader and find a post
2. Tap on the Visit link to see the post full screen.
3. Find the like button and tap on it.

You should be presented with a login screen. If not, clear all the cookies first. There's a helper for the debugger `__removeAllWordPressComCookies()`
